samples/AlarmBot/Responses/ResponseHelpers.cs

```diff
@@ -35,6 +35,5 @@ public static IMessageActivity ReplyWithTitle(this IBotContext context, string t
 
             return ((Activity)context.Request).CreateReply(sb.ToString());
```

Comment:
The (Activity) cast could be removed here.

Reply:
Good eyes! Will update it